projects/igniteui-angular/src/lib/core/styles/components/splitter/_splitter-theme.scss

```diff
@@ -182,10 +182,12 @@
 
         &::before {
             @extend %igx-splitter-hitbox;
-            top: calc(100% - #{map-get($theme, 'size')});
-            left: calc(100% - #{map-get($theme, 'size') * 2});
-            width: #{map-get($theme, 'size') * 4};
-            height: #{map-get($theme, 'size') * 3};
+            @if type-of(map-get($theme, 'size')) == 'number' {
```

Comment:
This check is actually not valid. What if the `size` prop refers to a CSS variable? The CSS `calc` function should work with CSS variable references. If they pass a bogus value, the function will simply fail, but the build shouldn't throw.